src/sagarY/whileLoop/WhileEx2.java

```diff
@@ -0,0 +1,24 @@
+package sagarY.whileLoop;
+
+class WhileEx2 {
+	int count = 0;
+	int sum = 1;
+	int targetNum;
+
+	void findRequiredNumbers(int targetNum) {
+		while (sum < targetNum) {
+			for (int index = 1; index < targetNum; index++) {
```

Comment:
loop inside loop not require

Reply:
int count = 0;
		int sum = 0;
		int num = 1 ;
		while(sum<targetNum) {
			sum = sum + num ;
			num++;
			count++;
		}